src/apurvaJun20/ReverseNo.java

```diff
@@ -0,0 +1,31 @@
+package apurvaJun20;
+
+import java.util.Scanner;
+
+public class ReverseNo {
+	static void reverseMethod(int number) {
+	       if (number < 10) {
+		   System.out.println(number);
+		   return;
+	       }
+	       else {
+	           System.out.print(number % 10);
+	           //Method is calling itself: recursion
+	           reverseMethod(number/10);
```

Comment:
Where is the logic ???
void findReverseNumber(int number) {
		int reverseNum = 0;
		int mod =0;
		while(number>0) {
			mod = number%10;
			number = number/10;
		    reverseNum = mod + reverseNum*10 ;
		}
		System.out.println("Reverse Number is "+reverseNum);
	}